arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-dialog.tsx

```diff
@@ -143,22 +116,118 @@ export class IDEUpdaterDialog extends AbstractDialog<UpdateInfo> {
       Widget.detach(this.widget);
     }
     Widget.attach(this.widget, this.contentNode);
+    this.appendInitialButtons();
     super.onAfterAttach(msg);
-    this.update();
+  }
+
+  private clearButtons(): void {
+    while (this.controlPanel.firstChild) {
+      this.controlPanel.removeChild(this.controlPanel.firstChild);
+    }
+    this.closeButton = undefined;
+  }
+
+  private appendNotNowButton(): void {
+    this.appendCloseButton(
+      nls.localize('arduino/ide-updater/notNowButton', 'Not now')
+    );
+    if(this.closeButton) {
```

Comment:
Please install [vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) in Code, and it will automatically format your code on save.

Reply:
I already have it installed, I don't know why it didn't work when I saved this one. Anyway, I'm fixing it, thanks.